docs/Configuration.md

```diff
@@ -12,6 +12,7 @@ DIGITS uses environment variables for configuration.
 | `DIGITS_JOBS_DIR` | ~/digits-jobs | Location where job files are stored. Default is `$DIGITS_ROOT/digits/jobs`. |
 | `CAFFE_ROOT` | ~/caffe | Path to your local Caffe build. Should contain `build/tools/caffe` and `python/caffe/`. If unset, looks for `caffe` in PATH and PYTHONPATH.|
 | `TORCH_ROOT` | ~/torch | Path to your local Torch build. Should contain `install/bin/th`. If unset, looks for `th` in PATH. |
+| `TENSORFLOW_PYTHON` | /usr/bin/python | Path to the python executable you would like to use for invoking and importing tensorflow. |
```

Comment:
If so, that would address my other comments about `sys.executable` vs. `config_value('tensorflow')['executable']`.